app/routes/_auth.tsx

```diff
@@ -1,19 +1,21 @@
-import { Box, Container } from '@chakra-ui/react';
+import { Flex, Container } from '@chakra-ui/react';
 import { Outlet, isRouteErrorResponse, useRouteError } from '@remix-run/react';
 import SeenErrorLayout from '~/components/errors/seen-error-layout';
 import UnseenErrorLayout from '~/components/errors/unseen-error-layout';
 import Header from '~/components/header';
+import Footer from '~/components/footer';
 
 export default function Index() {
   return (
-    <Box>
+    <Flex flexDir="column">
       <Header />
       <main>
-        <Container maxW="container.xl" px={{ base: 4, md: 45 }}>
+        <Container flex="1" maxW="container.xl" px={{ base: 4, md: 45 }} minHeight="100vh">
```

Comment:
For `flex="1"` to work, you need to put it on a direct child of the `<Flex>` parent.  Can you eliminate `main`?  If not, use `<chakra.main flex="1">`